addons/web/static/tests/webclient/switch_company_menu.test.js

```diff
@@ -361,3 +361,29 @@ test("companies can be logged in even if some toggled within delay", async () =>
     expect(".dropdown-menu").toHaveCount(0, { message: "dropdown is directly closed" });
     expect(["2"]).toVerifySteps();
 });
+
+test("always show the name of the company on the top right of the app", async () => {
+    const customSetup = () => {
+        // initialize a single company
+        patchWithCleanup(session.user_companies, {
+            allowed_companies: {
+                1: { id: 1, name: "Single company", sequence: 1, parent_id: false, child_ids: [] },
+            },
+            disallowed_ancestor_companies: {},
+            current_company: 1,
+        });
+    };
+    customSetup();
+
+    function onSetCookie(key, values) {
+        if (key === "cids") {
+            expect.step(values);
+        }
+    }
+    await createSwitchCompanyMenu({ onSetCookie });
+    expect(["1"]).toVerifySteps();
+
+    // in case of a single company, drop down button should be displayed but disabled
+    expect(".dropdown-toggle").toBeDisplayed();
```

Comment:
The feature is to show the name of the company... Would be nice to assert it.
expect(".dropdown-toggle").toHaveText("Single company");